ui/src/audioplayer/Player.js

```diff
@@ -37,10 +37,24 @@ const useStyle = makeStyles(
       },
     },
     qualityInfo: {
-      marginTop: '-2px',
+      marginTop: '-4px',
+      opacity: 0,
+      transition: 'all 500ms ease-out',
+    },
+    audioName: {
+      '&:hover + $qualityInfo': {
+        opacity: 1,
+      },
     },
     player: {
       display: (props) => (props.visible ? 'block' : 'none'),
+      '& .music-player-panel': {
+        height: '100px',
+        paddingBottom: '10px',
```

Comment:
I think making the player taller is worse than having the link appearing on top of the progress bar when you are actually hovering a link. I prefer to not do this.

Reply:
Alright, so it's okay that the link appears on top of the progress bar?

Reply:
Yes, it is ok.

IMHO, if you are hovering a link, it is expected you will have an address showing in the bottom, and you are probably not interested in seeing the progress bar at that moment.

Reply:
Yeah, that makes sense, I'll remove it and commit